.gitignore

```diff
@@ -18,6 +18,8 @@
 /testem.log
 /yarn-error.log
 .DS_Store
+.vscode
```

Comment:
This was annoying me but I'd be happy to pull this out to a separate PR if you don't think it fits here?

Reply:
no that's completely fine 🎉